psi4/src/export_cppe.cc

```diff
@@ -0,0 +1,73 @@
+/*
+ * @BEGIN LICENSE
+ *
+ * Psi4: an open-source quantum chemistry software package
+ *
+ * Copyright (c) 2007-2018 The Psi4 Developers.
+ *
+ * The copyrights for code used from other parties are included in
+ * the corresponding files.
+ *
+ * This file is part of Psi4.
+ *
+ * Psi4 is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * Psi4 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with Psi4; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * @END LICENSE
+ */
+
+#include "psi4/pybind11.h"
+
+#include "psi4/libmints/basisset.h"
+#include "psi4/libmints/matrix.h"
+#include "psi4/libpsipe/psipe.h"
+
+using namespace psi;
+#ifdef USING_cppe
+
+void export_cppe(py::module& m) {
```

Comment:
@loriab What do you think about name spacing such extensions under `core.cppe.*`?

Reply:
I haven't thought about such matter, but it feels sensible. So cautious yes. Though I don't think we want to go explicit subpackage, `import psi4.core.cppe` with separate compiled lib, as @robertodr was having some problems with structuring that.

Reply:
@loriab I managed to solve the problem, so it can be done without too much effort.